srcpkgs/nicotine+/template

```diff
@@ -1,16 +1,15 @@
 # Template file for 'nicotine+'
 pkgname=nicotine+
-version=1.4.1
+version=2.0.0
 revision=1
-wrksrc="nicotine-plus-${version}"
 archs=noarch
-build_style=python2-module
-hostmakedepends="python-gobject2-devel"
-depends="pygtk mutagen librsvg"
-pycompile_module="pynicotine"
+wrksrc="nicotine-plus-${version}"
+build_style=python3-module
+hostmakedepends="python3"
+depends="python3-gobject python3-mutagen gtk+3 librsvg"
 short_desc="Soulseek music-sharing client written in Python"
 maintainer="doggone <doggone@airmail.cc>"
 license="GPL-3.0-or-later"
-homepage="http://www.nicotine-plus.org/"
+homepage="https://nicotine-plus.github.io/nicotine-plus"
```

Comment:
Even though the nicotine-plus.org domain now redirects, https://nicotine-plus.org/ is the preferred URL.